chemprop/sklearn_predict.py

```diff
@@ -57,7 +57,7 @@ def predict_sklearn(args: SklearnPredictArgs) -> None:
     avg_preds = avg_preds.tolist()
 
     print(f'Saving predictions to {args.preds_path}')
-    assert len(data) == len(avg_preds)
+    # assert len(data) == len(avg_preds)
```

Comment:
If we want to eventually revisit this with a unit test, we could leave a TODO comment here like you did with the other changes.